README.md

```diff
@@ -58,9 +58,44 @@ iptables -t nat -A PREROUTING -i eth0 -p tcp -d 172.4.0.1 --dport 1883 -j DNAT -
 iptables -t nat -A PREROUTING -i eth0 -p tcp -d 172.4.0.1 --dport 8883 -j DNAT --to 192.168.1.10:8883
 ```
 
+Please note that you can also have the container running in your local network and will not need to modify IPTables. For example if you local network is on 192.168.1.x and your container's IP address is 192.168.1.10, point your local DNS resolver to that IP address and you do not need to add iptable entries.
+
 # Running the app
 
-You will need to run some form of SSL proxy (NGINX, Traefik, etc.) in front of this service. 
+You will need to run some form of SSL proxy (NGINX, Traefik, etc.) in front of this service.
+The SSL service is required and will need to listen on 443 and reverse proxy back to the docker container service on port 5000.
+You will need to generate a self-signed certificate for it. It can be generated with openssl:
+
+...
```

Comment:
You can use Markdown for this. Replace ... with \`\`\`sh and at the end replace ... with \`\`\`
```suggestion
```bash
```